wsmaster/che-core-sql-schema/src/main/resources/che-schema/6.16.0/2__create_workspace_activity_table.sql

```diff
@@ -0,0 +1,30 @@
+--
```

Comment:
Do not forget to update migration scripts names after merging of https://github.com/eclipse/che/pull/12222

Reply:
I wanted to merge #12222 only after merging this one.

Reply:
Yes, I understand. It was just reminding to rename your `2__`, `3__` files to `3__`, `4__` after merging #12222

Reply:
Ah, true. :+1: